### DISCUSSION TITLE
Resize only works partially in Safari

### DISCUSSION THREAD

**Discussion by flack** [Studio-42/elFinder#3310] [Ideas]
Tested on https://studio-42.github.io/elFinder/

when I make elfinder smaller in Firefox, it looks like this:
![Bildschirmfoto 2019-06-19 um 20 27 52](https://user-images.githubusercontent.com/425166/59790704-d7a0c000-92d0-11e9-9d39-4ee987732d97.png)

when I make it smaller in Safari, it looks like this:
![Bildschirmfoto 2019-06-19 um 20 28 01](https://user-images.githubusercontent.com/425166/59790729-e4bdaf00-92d0-11e9-80b4-cbf2423280dc.png)

i.e. it doesn't make the columns smaller, they stay the same and you get a horizontal scrollbar. 

However, making it bigger works the same in both browsers, i.e. in Safari the columns become bigger, too

**Comment by nao-pon** [comment]
@flack Did you fix (change) the column width in safari? Please reset the settings in your personal settings and try.

![FireShot Capture 036 - Test here_elFinder - Web File Manager - open source project under the_ - studio-42 github io](https://user-images.githubusercontent.com/1412630/61116732-fb21e980-a4cf-11e9-867d-7b1406d1c332.png)

**Comment by flack** [comment]
@nao-pon yes, doing the reset temporarily fixed the problem. However, as soon as I resize a column after, it gets stuck again. While in Firefox I can resize the columns, and later, when I resize the window, the columns are still responsive

**Comment by nao-pon** [comment]
I do not have an environment that can be debugged with Safari, so I can not examine it in detail.

We will widely seek information and PR to solve this problem.

**Comment by flack** [comment]
@nao-pon I did a `git bisect` on the 2.1 release branch, and according to that, the bug was introduced in this revision: a2a3e37e5b4dd014926f5dc6c2a198d52c89b3a9. This is the first revision where the columns overflow the elFinder window (i.e. they are not resized properly). And then later, starting from revision
c4e1d61b12c52e3b1037a0203d54a26f3af5c083, a scrollbar is shown. Hope this helps
